templates/modcontrol/modcontrol.html

```diff
@@ -2,6 +2,9 @@
 
 <div id="modcontrol_content" class="content" style="padding-top:2em;">
   <div class="form skinny">
+    <p style="font-size:1.3em"><a href="/modcontrol/auditlog">Audit Log</a></p>
```

Comment:
Inline styles are bad. Maybe we can fix that for the rest of this page? (Plus, these aren’t paragraphs. Well, if not, at least use a semicolon for consistency.)